egs/librispeech/ASR/pruned_transducer_stateless7/optim.py

```diff
@@ -75,20 +75,28 @@ def batched_params(self, param_group):
         batches = defaultdict(
             list
         )  # `batches` maps from tuple (dtype_as_str,*shape) to list of nn.Parameter
+        batches_names = defaultdict(
+            list
+        )  # `batches` maps from tuple (dtype_as_str,*shape) to list of str
 
-        for p in param_group:
+        for p, named_p in zip(param_group, group_params_names):
```

Comment:
What if `group_param_names` is `None`?

Reply:
It could not be `None`.  The most import part of this pr is tracking name of each parameter.

Reply:
The default value was None.